Tryouts/Plugins/ApplicationPlugins/SimpleNodeApp/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "simple-node-app",
+  "version": "0.0.0",
+  "description": "SimpleNodeApp",
+  "main": "app.js",
+  "author": {
```

Comment:
This node application can't run, because you have no execution step here. Right now it can only run in visual studio as a node.js project.

Reply:
this is essentially not a node application - it is started by the browser. It does not need to be build - only installed JQuery package. I am just not sure how to prevent it from building, but make it still install the package.

Reply:
You aren't creating a bundle from this package and - unless I haven't spotted it - nothing serves this demo application.
So essentially an html file gets loaded to the webview over the file:// protocol.

Reply:
yes, exactly - but there is a twist to it - it needs to unpack the node JQuery package.